src/pages/_data/navigation/mainNav.json5

```
@@ -6,8 +6,9 @@
     },
     { title: "Get Building", url: "/get-building/", children: [
         { title: "Distribute your App", url: "/get-building/distribute-your-app/" },
-        { title: "Advanced Installation Guide", url: "/get-building/install-advanced/" },
-      ] 
+        { title: "Advanced Holonix Guide and Nix Knowledge", url: "/get-building/install-advanced/" },
+        { title: "Develop without Nix (currently unsupported)", url: "/get-building/install-without-nix/" },
```

People will have to know what Nix means, so an advanced level is implicit IMO. That it's unsupported is the first disclaimer on the page then. To keep it short,
```suggestion
        { title: "Setup without Nix", url: "/get-building/install-without-nix/" },
```